k8s/deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: promotions
+  labels:
+    app: promotions
+spec:
+  replicas: 2
```

Comment:
do we need to replicas for now ? we can set it to 1 for simplicity, how do you think ?

Reply:
Yes that makes sense.